src/main/main.c

```diff
@@ -234,7 +234,7 @@ void init(void)
     servoMixerInit(masterConfig.customServoMixer);
 #endif
 
-#ifdef SPEKTRUM_BIND
+#if defined(SERIAL_RX) && defined(SPEKTRUM_BIND)
```

Comment:
What if the target does not want to have X defined? No way to #undef X then. 
This "sanitation.h" might work if we did NOT use "#if defined X" but rather "#if (USE_X = "YES") instead. I.e. all conditionals tested on value, not only on existence.